### PR TITLE
feat: add integration for exception queue endpoint.

### PR DESCRIPTION
**Pull request by uzairbaig**

### CODE REVIEW

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java [hunk 1]
```diff
@@ -87,7 +89,7 @@ public void configure() {
         .unmarshal().json(JsonLibrary.Jackson);
 
     from("direct:tcs-connection")
-        .setHeader(OIDC_ACCESS_TOKEN_HEADER).method(keycloakBean, GET_TOKEN_METHOD)
+        //.setHeader(OIDC_ACCESS_TOKEN_HEADER).method(keycloakBean, GET_TOKEN_METHOD)
```
Comment (DorisWongNHS) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java:92]: Will need to uncomment this one

Comment (jayanta2018) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java:92]: Do you want this comment here? @uzairbaig

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java [hunk 2]
```diff
@@ -121,5 +123,21 @@ public void configure() {
         .setHeader(Exchange.HTTP_METHOD, constant(HttpMethod.POST))
         .setHeader(Exchange.CONTENT_TYPE, constant(MediaType.APPLICATION_JSON))
         .toD(serviceUrlConnection + API_CONNECTION_REMOVE);
+
+    //direct:connection-exception-summary
```
Comment (jayanta2018) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java:127]: and this one?

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java (hunk 1 above)
Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java]: fixed.

Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java]: fixed.

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java (hunk 2 above)
Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/ConnectionServiceRouter.java]: fixed.

**Review by DorisWongNHS — APPROVED** [review]

**Pull request merged**